samples/Accelerometer.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2016, Intel Corporation.
+
+// Test code to use the AccelerometerSensor (subclass of Generic Sensor) API
+// to communicate with the BMI160 inertia sensor on the Arduino 101
+// and obtaining information about acceleration applied to the X, Y and Z axis
+console.log("Acclerometer test...");
+
+
+var sensor = new AccelerometerSensor({
```

Comment:
AccelerometerSensor -> Accelerometer
Just a heads up, very soon, we will split the Accelerometer interface to 2 interfaces, one for Accelerometer and another for LinearAccelerometer. In that case includeGravity will not be needed. You may want to change this when it is reflected in the spec.